specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -312,8 +315,7 @@
           "200": {
             "description": "OK -- Update Share operation completed successfully.",
             "schema": {
-              "$ref": "#/definitions/FileShare",
-              "description": "Properties of the updated file share."
```

Comment:
We should not remove the description, or the doc/SDK generated from swagger will missing description for this property.
We can change it to make it more accurate.